src/plugins/data_view_field_editor/public/service/format_editor_service.ts

```diff
@@ -52,7 +52,7 @@ export class FormatEditorService {
       truncateFormatEditorFactory,
       urlFormatEditorFactory,
       histogramFormatEditorFactory,
-    ];
+    ] as Array<FieldFormatEditorFactory<unknown>>;
```

Comment:
I'd just like to verify that this is an improvement. It seems to me that `:` notation is preferred since it won't cover up incorrect types. I'm not sure if passing `unknown` is doing anything.

Reply:
This casting is to make the array single-typed, but you are right - it covers up incorrect types. 

This stays with the `:` notation, but I'm still stuck on how to completely remove `any` from the file: 798f29d1959